saw-remote-api/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM haskell:8.8.4 AS build
 USER root
-RUN apt-get update && apt-get install -y wget libncurses-dev unzip
+RUN apt-get update && apt-get install -y wget libncurses-dev libreadline-dev unzip
```

Comment:
This stage of the `Dockerfile` is only used for building `saw-remote-api`, and the results are copied into a later stage that is actually used when running a container. That is to say: you should get installing `libreadline-dev` in the second invocation of `apt-get install`, not the first one.

Reply:
got it!